modules/tinymce/src/themes/silver/main/ts/ui/header/InlineHeader.ts

```diff
@@ -24,6 +24,8 @@ export interface InlineHeader {
 
 const { ToolbarLocation, ToolbarMode } = Options;
 
+const scanIfWithinDistance = 40;
```

Comment:
The name of this variable is a verb. I think it would be more appropriate to name it with a name indicative of what the variable is used for, like `minimumDistanceToChange`

Reply:
A good name is a bit tricky, though I agree with your point.